src/screens/Home/index.tsx

```diff
@@ -1,16 +1,21 @@
 import React from 'react';
+import { useEffect } from 'react';
```

Comment:
```suggestion
import React, { useEffect } from 'react';
```

They are from the same module. We should group them up